### DISCUSSION TITLE
API Version 08-01 Deprecation

### DISCUSSION THREAD

**Discussion by Bahri-19** [microsoft/sample-app-aoai-chatGPT#545] [General]
Hi, I use an adjusted version of this repo for multiple projects running in prod, I received an email from Microsoft that the API Version 08-01-2023-Preview will be deprecated in April. Will there be any update for this repo that uses the API version 12-01-2023-Preview. 

Thank you for the ongoing efforts.

**Comment by superbillen** [comment]
I recieved the same mail and have the same question?
I do not know how to do this update manually
Bahri-19 do you have any information
